README.md

```diff
@@ -15,51 +15,77 @@ Releases        | <https://github.com/DiamondLightSource/odin-fastcs/releases>
 
 ## Development
 
-1. Clone odin-data and open in its devcontainer (for now, checkout fastcs-dev branch)
-2. Build odin-data into `vscode_prefix`
+In most cases it should be sufficient to work within the odin-fastcs devcontainer (or a
+virtualenv) and run the unit tests to develop new features and bug fixes.
+
+In order to develop and test odin-fastcs alongside local dev versions of odin-control
+and odin-data applications, a development environment can be run up from the odin-data
+devcontainer.
+
+### Full Development Environment
+
+The following instructions assume use of vscode, as it makes it very convenient to set up
+an environment that "just works". However, it is possible to run the same environment up
+manually using the [odin-data-developer] container and the following steps should still
+provide a good overview.
+
+1. Clone [odin-data] (next to odin-fastcs), checkout the `fastcs-dev` branch and re-open
+   in devcontainer
+2. Add odin-fastcs to the workspace using `Workspaces: Add Folder to Workspace...`
+3. Build odin-data into `/odin`:
 
     i. `CMake: Delete Cache and Reconfigure` (select a compiler, /usr/bin/gcc is
     probably best)
 
     ii. `CMake: Install`
 
-3. `Workspaces: Add Folder to Workspace...` to add odin-fastcs to the workspace
 4. Install odin-fastcs and its odin dev environment
 
-    i. `pip install -e .[dev,odin]`
+    i. `Python: Select Interpreter` and set it to `/venv/bin/python` for the workspace
 
-    ii. `Python: Select Interpreter` and set it to `/venv/bin/python` for the workspace
+    ii. `pip install -e .[dev,odin]`
 
 5. Prepare the dev environment
 
     i. `dev/configure.sh one_node_fp /workspaces/odin-data/vscode_prefix /venv`
 
 6. Run the dev environment
 
-    i. `dev/start.sh` (it may print some garbage to the terminal while it installs)
```

Comment:
You can avoid the garbage with
```
mkdir ~/.config/zellij
zellij setup --dump-config > ~/.config/zellij/config.yaml
```

Reply:
This doesn't avoid the installing WASM step and I can't find a way that doesn't leave a terminal open, so I have just added an instruction to run zellij once and let it install.